### PR TITLE
chore: fix Eslint issues

### PR DESCRIPTION
**Pull request by christian-bromann**
## Problem

My IDE reported some Eslint inconsistencies.

## Solution

Fixed them.

### CODE REVIEW

File: tests/screenobjects/components/carousel.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Carousel extends Gestures {
         const expectedText = partialText.toLowerCase();
 
         if (browser.isIOS) {
-            return expect(cardText).toContain(expectedText);
+            return global.expect(cardText).toContain(expectedText);
```
Comment (wswebcreation) [inline tests/screenobjects/components/carousel.js]: Hmm, in the `browser.dynamic.spec.js` it's not a global. Personally I not fond of the global, can we make this generic over all files (remove global, add it to the eslint)?

**Comment by wswebcreation** [comment]
Hi @christian-bromann 

Tnx for fixing this, do you have time to pick up the review comment, or should I do it?

**Comment by christian-bromann** [comment]
Fixed comment, also applied change from #6

**Comment by wswebcreation** [comment]
Tnx man!

**Pull request merged**